react-hot-button-component/my-app/src/HotButton.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+
+export default function HotButton () {
+  const allColors = ['#000000', '#351C74', '#674DA7', '#E06666', '#F6B26A', '#FEFF00', '#FFFFFF'];
+  let [numClicks, setNumClicks] = useState(0);
```

Comment:
Get in the habit of _always_ using `const` for `useState` variables. That will keep you from accidentally setting its value and thinking it might actually change! (Well, it will change but only for this single render.)